api/libproject/devstart.rb

```diff
@@ -117,21 +134,17 @@ def clean()
 })
 
 
-def run_api_and_db(*args)
+def run_api_and_db()
   common = Common.new
-  account = get_auth_login_account()
-  if account == nil
-    raise("Please run 'gcloud auth login' before starting the server.")
-  end
   common.status "Starting database..."
   common.run_inline %W{docker-compose up -d db}
-  run_api(account)
+  run_api()
 end
 
 Common.register_command({
   :invocation => "run-api",
   :description => "Runs the api server (assumes database and config are already up-to-date.)",
-  :fn => lambda { |*args| run_api_and_db(*args) }
+  :fn => lambda { |*args| run_api_and_db() }
```

Comment:
Interesting tidbit: Codacy told me that this is preferred: `:fn => -> (*args) { run_api_and_db() }` but I'm not yet sure if I agree.

Reply:
Yeah, I have no preference here... except that maybe we should turn off Codacy lint rules. :)